### ISSUE TITLE
Fuzzy Crafting Table is not fuzzy at all

### ISSUE THREAD

**Issue by AT611** [RS485/LogisticsPipes#444]
First thing: I'm aware of #381.
I have somewhat similar issue with Logistics Fuzzy Crafting Table and Logistics Crafting Pipe with Fuzzy Crafting Upgrade on it, but with a fixed output.
I'm trying to make it to craft the Wood Gear from Buildcraft. I've set it up as a plain recipe in the fuzzy crafting table (4 vanilla sticks equals 1 wood gear) and imported the recipe from it into the crafting pipe, changing the mode on the sticks to the oredict one. When I request for a wood gear, sticks from Natura are landing into the crafting table, but no crafting were made. If after that I take out natura sticks from the crafting table inventory and put in vanilla sticks instead - crafting continues.
So the Fuzzy Crafting Table works the exact same way as a non-fuzzy one.

**Comment by SinusoidalC** [comment]
@cybcaoyibo you should read this

**Comment by AT611** [comment]
Update:
After some more looking around I've found the original PR #363, in which there were a screenshot, that illustrated Fuzzy Crafting Upgrade functionality in the Logistics Fuzzy Crafting Table. I was not aware of that functionality, so I guess my issue is now invalid. But still: that pop-up menu appeared only after several seconds (not less!) of holding the mouse cursor above an item. Really confusing. I would have noticed the menu if it was as quick as in Crafting Logistics Pipe with that Fuzzy Crafting Upgrade in it.
My problem is solved anyway, but that pop-up delay is just wrong: at first glance I saw no delay in the PR code, so I guess that delay is a bug.
